### DISCUSSION TITLE
Flipper alternatives

### DISCUSSION THREAD

**Discussion by Phreak87** [djsime1/awesome-flipperzero#56] [General]
If you search for some Flipper alternatives (based on FAQ build own Flipper) i found a project that is in many cases a better alternative (bigger colored display, Speed, memory, cheaper): Its a la104 Logic analyzer. with the modified gabonator firmware you can do nfc, ir and radio hacking - and a loooooot more (all based on extension boards like Flipper zero). but ... Its not active as flipper zero ...

**Comment by RChadwick7** [comment]
I have the LA104 with Gabonator. It's pretty awesome in potential. Unfortunately, nobody ever ran with it. Definitely a better screen, more useable space. No NFC or RFID built in but I'm sure it could be added. I made the RF board and it worked well as a Flipper-like spectrum analyzer.

Sep 17, 2022 7:25:17 PM Markus Eckl ***@***.***>:

**Reply by Phreak87** [reply]
Maybe with this post somebody is intersted in this and do a fork with active development. If there was a active dev behind im sure this could be a more powerful alternative - especially because all apps are handled as plugin.

**Comment by djsime1** [comment]
Never heard of the LA104 until now, but it's definitely an interesting device. I would consider it a semi-alternative at best, since it feels like the alternate firmware is headed in a slightly different direction than Flipper. I know you didn't ask, but for future reference I probably won't include 'Flipper alternatives' in this repo; kind of defeats the whole point of it being an Awesome List.
